src/hooks/useGet.js

```diff
@@ -22,6 +22,7 @@ export default function useGet({
   url,
   data,
   params,
+  method = 'get',
```

Comment:
I don't love a function called `useGet` that is not making a GET request.

Reply:
cc @Atticus29 yeah it's the worst but elasticsearch requests for data function exactly like GET requests but for weird reasons use the POST method instead. open to other suggestions but this felt clean and not-toooo-dirty to me

Reply:
If this is only going to be used for `useGetMergeConflicts`, then my preference would be to just duplicate the logic in that function.

If this is required for all elasticsearch interaction, then maybe create a new function `useElasticSearchGet` and either duplicate the logic or abstract and share it.

If you prefer to leave it as is, I prefer renaming the function to remove the word 'Get'.

Reply:
okay it is switched to `useFetch` everywhere. was this worth the time? probably not. but it's done now.